src/roadmap/releases.md

```diff
@@ -93,12 +93,6 @@ deisrel changelog individual $COMPONENT $OLD_RELEASE $NEW_SHA $NEW_RELEASE | pbc
 open https://github.com/deis/$COMPONENT/releases/new?tag=$NEW_RELEASE
 ```
 
-### Step 4: Publish Release to Versions Server
```

Comment:
@mboersma the subsequent Workflow Chart Release steps?  They appear to start fresh from `1.`, right?